docusaurus/docs/nodejs-builtin-fallbacks.md

```diff
@@ -0,0 +1,66 @@
+---
+id: nodejs-builtin-fallbacks
+title: NodeJS builtin fallbacks
+---
+
+NodeJS builtin fallbacks enable you to import NodeJS builtin modules meant for Node and fallback to browser specific modules in your web application.
+
+Per default Create React App set fallbacks to empty modules in production build and development fallbacks in development mode.
+
+In development mode you might get error messages in your browser console similar to:
```

Comment:
```suggestion
In development mode, you receive error messages in the terminal where you're running the development server similar to:
```

i'm seeing the errors in my terminal where i'm running `npm start`, not browser console